### PR TITLE
[feat]finished code but tailwindcss isnt working properly for me so idk if it actually works T_T

### PR DESCRIPTION
**Pull request by saitunguturu**
I am confused because I'm using it in Adopt and it isn't an issue there. maybe installation issue? i need help troubleshooting later.

[//]: # "Feel free to customize this template to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- queried data from the public Posts table on Supabase
- displays the content on localhost:3000/supabase

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 
image
It looks like that on my screen because tailwindcss isn't properly installed. Hopefully it looks better on your end?

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- please review Post for each individual post's format
- PostList displays all of the posts

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- install tailwindcss properly
- refine my code

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @kylezryr

### CODE REVIEW

File: package.json
Comment (kylezryr) [inline package.json:1]: i found the issue with tailwind - for some reason the install command installed an old version of tailwind where our setup didnt work. you can fix this with running `pnpm install -D tailwindcss@latest postcss@latest @tailwindcss/postcss@latest` to install the newest version of the dependencies we need. **you then also need to add this line to the top of app/layout.tsx:** 

```
import '../assets/global.css';
```

sorry about that, not sure why this happened!

File: components/Post.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    year: 'numeric',
+  });
+
+  return (
+    <div
+      key={id}
+      className="bg-white border border-gray-200 rounded-xl p-5 shadow-sm space-y-4 max-w-md w-full hover:shadow-md transition-shadow duration-200"
+    >
+      {/* Header */}
+      <div className="flex flex-col">
```
Comment (kylezryr) [inline components/Post.tsx:30]: i think you're missing the profile pciture icon here like we used to have

File: components/Post.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            className="w-full max-h-64 object-cover rounded-md"
+          />
+        </div>
+      )}
+
+      {/* Text */}
+      <p className="text-gray-800 text-sm leading-relaxed">{text}</p>
+
+      {/* Footer */}
+      <div className="text-sm text-gray-600">❤️ {likeCount} likes</div>
```
Comment (kylezryr) [inline components/Post.tsx:53]: you should use the previous `HeartIcon` here, and also include the `ShareIcon`!

File: components/PostList.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { useEffect, useState } from 'react';
+//imports Post compnent and the props I need to store the Post data
+import Post, { PostProps } from '@/components/Post';
+import { supabase } from '@/lib/client';
+
+export default function PostList() {
+  //sets consts
+  const [posts, setPosts] = useState<PostProps[]>([]);
+
+  async function fetchPostData() {
```
Comment (kylezryr) [inline components/PostList.tsx:12]: this query looks good! lets move it to a `query.ts` file inside of `lib` folder so that we can import it and use it here instead, just to clean up and organize our code a little better

File: app/supabase/page.tsx
Comment (kylezryr) [inline app/supabase/page.tsx:1]: you can move this to be `app/page.tsx` once the tailwind is working

File: components/PostList.tsx
Comment (kylezryr) [inline components/PostList.tsx:1]: this component is good but may not be necessary, since we can just put the mapping of the posts directly in the page itself instead of placing it in a component. generally, if we don't reuse a component (since we only render `AllPosts` once in the page), its probably fine to just place it in the page itself instead of making a new component!

**Review by kylezryr — CHANGES_REQUESTED** [review]
nice work Sai! sorry about the tailwind issues, i left a comment on how to fix it. looks good otherwise, just a couple of changes to make! if u have the time, feel free to attempt the comments stretch question too
